### PR TITLE
SUPPORT RTL/LTR ON LOGIN PAGE

### PR DESCRIPTION
**Pull request by ahmedrowaihi**

### CODE REVIEW

File: src/Pages/Login.tsx [hunk 1]
```diff
@@ -154,22 +154,23 @@ export default function LoginPage() {
           <div className="logo" onClick={() => navigate("/")}>
             Snort
           </div>
-          <h1>
+          <h1 dir="auto">
```
Comment (verbiricha) [inline src/Pages/Login.tsx:157]: We can apply it to this whole container at once: `<div className="login-container" dir="auto">` and should work on all items.

Comment (ahmedrowaihi) [reply src/Pages/Login.tsx:157]: I have tried but seems not applying to all elements.

You can test and confirm 👍

Comment (verbiricha) [reply src/Pages/Login.tsx:157]: Interesting, wonder why that is. One minor nitpick: could we add `margin-right: 25px` after https://github.com/v0l/snort/blob/main/src/Pages/Login.css#L40 so it looks the same in rtl? Other than that lgmt, thanks for your contributions!

Comment (ahmedrowaihi) [reply src/Pages/Login.tsx:157]: Sure, that's perfect.
Glad to contribute 🙏🏼.

Comment (ahmedrowaihi) [reply src/Pages/Login.tsx:157]: Added @verbiricha

**Review by verbiricha — APPROVED** [review]

**Pull request merged**